packages/wdio-cli/src/commands/config.ts

```diff
@@ -65,6 +65,16 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false)
         if (!hasPackage('ts-node')) {
             packagesToInstall.push('ts-node', 'typescript')
         }
+        if (!hasFile('tsconfig.json')){
+            const config = {
+                'compilerOptions': {
+                    'types': ['node', 'webdriverio/async', frameworkPackage.package],
+                    'target': 'ES6'
```

Comment:
Let's keep es5 as a default please, having es6 will cause issues when writing unsupported code inside `execute` commands for Internet Explorer for example and the output doesn't matter other then compatibility.

Reply:
@erwinheitzman  i believe browsers support only es5 right ? if i am not wrong

Reply:
I have some doubts how stable this is. Babel doesn't know that the piece of function is executed in the browser. It mostly defines helper variables outside of it which won't be available when run in the browser.

Reply:
Babel? This is Typescript which will change your arrow functions for example to regular function (if your target is es5 instead of es6) which is needed for IE (specifically for `execute` command only).

Reply:
@praveendvd up to date browsers support higher then es5 but I can't find all the supported versions for each browser this minute

Reply:
all in all, this transforms the user code from typescript syntax to the chosen javascript output syntax.

Reply:
I see you changed it to es2018 which is even newer so that doesn't help :P

Reply:
to give you an idea `es6` === `es2015` as far as I am aware there's no `es2XXX` variant for `es5` ;)

Reply:
`lib` is used to allow the user to write specific syntax versions
`target` is used to compile to a specific javascript version for compatibility reasons